pkg/controller/controller_instance_test.go

```diff
@@ -2424,12 +2424,13 @@ func TestPollServiceInstanceFailureProvisioningWithOperation(t *testing.T) {
 	assertNumberOfActions(t, actions, 1)
 
 	updatedServiceInstance := assertUpdateStatus(t, actions[0], instance)
-	assertServiceInstanceProvisionRequestFailingErrorNoOrphanMitigation(
+	assertServiceInstanceRequestFailingErrorStartOrphanMitigation(
```

Comment:
@MHBauer we already had a test for this scenario, but because I thought that this was a valid behavior for OSB spec, I wrote a test to assert it as well (there was no bug in the code, the problem is OSB compliance).